src/features/defi/components/Deposit/Deposit.tsx

```diff
@@ -120,6 +120,8 @@ export const Deposit = ({
   const [activeField, setActiveField] = useState<InputType>(InputType.Crypto)
   const [percent, setPercent] = useState<number | null>(null)
   const amountRef = useRef<string | null>(null)
+  const bgColor = useColorModeValue('gray.50', 'gray.850')
+  const borderColor = useColorModeValue('gray.100', 'gray.750')
```

Comment:
Unrelated to this fix, but brought the hook up to the top so conditional render won't break react.